.pre-commit-config.yaml

```diff
@@ -1,15 +1,15 @@
--   repo: git://github.com/pre-commit/pre-commit-hooks
-    rev: a11d9314b22d8f8c7556443875b731ef05965464
-    hooks:
-    -   id: trailing-whitespace
-    -   id: flake8
-    -   id: check-yaml
-    -   id: check-added-large-files
-    -   id: debug-statements
-    -   id: end-of-file-fixer
-        exclude: '^.+?\.json.+?\.yml$'
--   repo: https://github.com/asottile/reorder_python_imports
-    rev: v0.3.5
-    hooks:
-    -   id: reorder-python-imports
-        language_version: python2.7
+-   repo: git://github.com/pre-commit/pre-commit-hooks
+    rev: a11d9314b22d8f8c7556443875b731ef05965464
+    hooks:
+    -   id: trailing-whitespace
+    -   id: flake8
+    -   id: check-yaml
+    -   id: check-added-large-files
+    -   id: debug-statements
+    -   id: end-of-file-fixer
+        exclude: '^.+?\.json.+?\.yml$'
+-   repo: https://github.com/asottile/reorder_python_imports
+    rev: v0.3.5
+    hooks:
+    -   id: reorder-python-imports
+        language_version: python2.7

```

Comment:
I don't see any actual changes in most of these lines. Was this an unintentional line ending change?

Reply:
I don't think we are doing linting for yaml files. If you didn't make any actual changes to this file, could you try reverting this file to its original state?

Reply:
It looks like we do [lint for yaml](https://github.com/learningequality/studio/pull/1085/files#diff-82c3b36709926de3a1b27bea0d8258ccR6) I ran it through a diff checker, and it says the two files are identical. I can't commit it because no changes are detected, but I think it should be okay to merge. Have no idea why git is saying the two are different